CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.8.0
+
+### Major changes
```

Comment:
Why don't to stop mixing up with terminology? Major changes could be only major release according to semver.